README.md

```diff
@@ -1 +1,51 @@
-# dat
+# Delta Acceptance Testing (DAT)
+
+The DAT project makes it easy to generate Delta Lake reference tables and run integration tests to check different implementations of the Delta Lake PROTOCOL behave as expected.
+
+The Delta Lake implementations can clone this repo, generate the reference tables, and implement the suggested integration tests in the language of their choice.
+
+## Installation
+
+You can install the dependencies for this project by running `poetry install`.  Run `poetry shell` to activate the virtual environment.    If `poetry shell` does not activate the environment on your machine, you may need to run this command:
+
+```
+source "$( poetry env list --full-path | grep Activated | cut -d' ' -f1 )/bin/activate"
+```
+
+Run `python -m dat.main` to make sure everything is working properly.
+
+## How to build tables
+
+Start by removing the `out` directory with `rm -rf out`.
+
+Create the reference tables with `python -m dat.main write-generated-reference-tables`.
```

Comment:
shouldn't this command just re the `out` folder automatically? let's make this one step, not two.

cc @edmondo1984

Reply:
@scottsand-db - are you asking if this command should automatically remove the `out` folder?  That'd make sense to me.

Reply:
@scottsand-db I would make it a parameter, (clean_before_write) and make it true as default, what do you think? So if for some reason one wants to execute without cleaning it can work

Reply:
@edmondo1984 that SGTM.

Reply:
@MrPowers aren't we going to update this to remove the `Start by removing the `out` directory with `rm -rf out`.` ?

Reply:
Sure, done